packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -0,0 +1,36 @@
+# psammead-storybook-helpers - [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-assets.svg)](https://www.npmjs.com/package/@bbc/psammead-assets) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+This package provides a collection of common values that are used in storybook by the Psammead components.
+
+## Exports
+
+`text-variants` - A list of text samples in different languages.
+`input-provider` - A function that provides support for selecting between the `text` and `select` storybook knobs.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-storybook-helpers --save-dev
+```
+
+## Usage
+
+```jsx
+import { LANGUAGE_VARIANTS } from '@bbc/psammead-storybook-helpers/text-variants';
+import { inputProvider } from '@bbc/psammead-storybook-helpers/input-provider';
+```
+
+## Contributing
+
+When **adding** a new export to this utility package the [export tests](https://github.com/bbc/psammead/blob/dab14a2732cfa620e083b7da66a148b4189474a7/packages/utilities/psammead-storybook-helpers/index.test.jsx#L7-L13) also need to be updated. When **removing** an exisiting export from this utility package the [export tests](https://github.com/bbc/psammead/blob/dab14a2732cfa620e083b7da66a148b4189474a7/packages/utilities/psammead-storybook-helpers/index.test.jsx#L7-L13) need to be updated and the package version requires a major change (EG: 1.2.1 -> 2.0.0) as this would be considered a breaking change due to functionality being removed.
```

Comment:
```suggestion
When **adding** a new export to this utility package, the [export tests](https://github.com/bbc/psammead/blob/dab14a2732cfa620e083b7da66a148b4189474a7/packages/utilities/psammead-storybook-helpers/index.test.jsx#L7-L13) also need to be updated. When **removing** an existing export from this utility package, the [export tests](https://github.com/bbc/psammead/blob/dab14a2732cfa620e083b7da66a148b4189474a7/packages/utilities/psammead-storybook-helpers/index.test.jsx#L7-L13) need to be updated and the package version requires a major change (EG: 1.2.1 -> 2.0.0) as this would be considered a breaking change due to functionality being removed.
```